composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^8.0",
-        "illuminate/contracts": "^9.0",
+        "illuminate/contracts": "^8.15|^9.0",
```

Comment:
However, there are no plans to support Laravel 8 right now. I don't think that there are enough people using it which cannot upgrade to L9.

Reply:
Alright, I understand